packages/parser/src/envelopes/apigw.ts

```diff
@@ -12,10 +11,7 @@ export const ApiGatewayEnvelope = {
     return Envelope.parse(APIGatewayProxyEventSchema.parse(data).body, schema);
   },
 
-  safeParse<T extends ZodSchema>(
-    data: unknown,
-    schema: T
-  ): ParsedResult<unknown, z.infer<T>> {
+  safeParse<T extends ZodSchema>(data: unknown, schema: T) {
```

Comment:
I'd consider putting back the return type, mostly for consistency with the other envelopes, but please dont' take this as a blocking comment if you don't want to.